src/power_grid_model_io/config/excel/vision_en.yaml

```diff
@@ -137,14 +146,15 @@ grid:
       r_grounding_to: Re2
       x_grounding_to: Xe2
       extra:
-        - Name
+        - Number
```

Comment:
transformer used to have Subnumber

Reply:
(and Node.Number)

Reply:
I think that's only for transformers created from the Transformer loads table.